admin/query/src/main/java/org/codice/ditto/replication/admin/query/replications/persist/UpdateReplication.java

```diff
@@ -24,15 +24,14 @@
 import org.codice.ddf.admin.common.fields.base.scalar.StringField;
 import org.codice.ddf.admin.common.fields.common.PidField;
 import org.codice.ditto.replication.admin.query.ReplicationUtils;
-import org.codice.ditto.replication.admin.query.replications.fields.ReplicationField;
 
-public class UpdateReplication extends BaseFunctionField<ReplicationField> {
+public class UpdateReplication extends BaseFunctionField<BooleanField> {
```

Comment:
I did this to avoid a timing issue where the Replication is saved, but not committed to Solr yet. This way clients will be forced to fetch via the GetReplications to get all the latest info.